data_diff/dbt.py

```diff
@@ -558,6 +560,8 @@ def _email_signup() -> None:
 
 def _extension_notification() -> None:
     if bool_notify_about_extension():
```

Comment:
I think we should rename these, but not blocking

Reply:
Cool. I'll merge this then!